src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorServerSide-CSharp/.template.config.src/template.json

```diff
@@ -0,0 +1,114 @@
+{
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "Blazor",
+    "ServerSide"
+  ],
+  "groupIdentity": "Microsoft.Web.Blazor.ServerSide",
+  "guids": [
+    "4C26868E-5E7C-458D-82E3-040509D0C71F",
+    "5990939C-7E7B-4CFA-86FF-44CA5756498A",
+    "650B3CE7-2E93-4CC4-9F46-466686815EAA",
+    "0AFFA7FD-4E37-4636-AB91-3753E746DB98"
+  ],
+  "identity": "Microsoft.Web.Blazor.ServerSide.CSharp",
+  "name": "Blazor (Server-Side in ASP.NET Core)",
```

Comment:
Super minor point, but the capitalization `Server-Side` looks odd to me.

Maybe: `Blazor (Server-side on ASP.NET Core)`

Reply:
Sure